ansible/playbook/openstack/README.adoc

```diff
@@ -28,18 +28,14 @@ NOTE: The list of flavors is identified on the link:../../../openstack/README.ad
 
 == Playbooks
 
-=== Create a VM
 === Create a VM
 
-Create OpenStack instance based on passwordstore
+Create OpenStack instance based on passwordstore.
```

Comment:
"Create OpenStack instance based on passwordstore." -> "Create an OpenStack instance (aka a VM) using passwordstore as tools to manage the credentials, information."

Reply:
Changed. Also added link to the `passwordstore` README.